com.wps.WPS.json

```diff
@@ -0,0 +1,104 @@
+{
+    "app-id": "com.wps.WPS",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "tags": ["proprietary"],  
+    "separate-locales": false,
+    "build-options": {
+        "no-debuginfo": true
+    },
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "command": "wps",
+    "cleanup": [
+        "/share/qt4",
+        "/lib"
+    ],
+    "modules": [
+        "shared-modules/qt4/qt4-4.8.7-minimal.json",
```

Comment:
You're not actually including the submodule correctly here

Reply:
should be fixed by https://github.com/flathub/flathub/pull/309/commits/52ef0644f04431f2124224438c0e97411a0ec9a1